maxim/tests/test_llamaindex.py

```
@@ -0,0 +1,257 @@
+        "condition": "Clear",
+        "humidity": 55,
+        "wind_speed": 15
+    }
+}
+
+dotenv.load_dotenv()
+
+baseUrl = os.getenv("MAXIM_BASE_URL")
+
+class TestLlamaIndex(unittest.TestCase):
+    def setUp(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)